.prettierrc

```diff
@@ -0,0 +1,11 @@
+{
+  "semi": true,
+  "trailingComma": "es5",
```

Comment:
Why not `true` here?

Reply:
These are the possible values https://prettier.io/docs/en/options.html#trailing-commas. I set it to `es5` since it seems to be the default. I could change it to `all`

Reply:
So I personally prefer `all` because re-ordering lists won't accidentally produce a situation where two array-like values are next to each other without a separating comma between them.

Reply:
A more practical example:

If there's a lot of imports from a module listed and I want to sort them but it's still a single line, I navigate to the import line in question, type `:s/\([{,]\)/\1\r/g` which instantly, correctly multi-lines the import statement, then I use `V` mode to highlight the imports and then type `:sort` to alphabetically sort them. Then in `V` mode again I highlight the whole import multi-line statement and hit `Shift+J` to immediately snap it back into a single line import again, but with re-ordered imports.

This doesn't work correctly when there isn't a trailing comma in the import, though, which is why you see me always re-inserting it in my PRs. (It's also why you see the re-ordering happening in the first place, it's a few seconds of typing per import that's not properly sorted.)